TODO_V2.md

```diff
@@ -1,5 +1,9 @@
 # TODO in mediasoup v2 (server-side)
 
+* `RtpPacket::RtxEncode()` should remove the original padding (also the header flag by using `SetPayloadPaddingFlag(false)`), then set the RTX payload, then check whether padding is required and, if so, set the padding flag and bytes.
```

Comment:
How does a RTX packet receiver suppose to know that padding is required on the original RTP?